docs/topic-guides/blendable-web-apps/starcounter-mvvm.md

```diff
@@ -32,11 +32,11 @@ Typed JSON allows dynamic creation of JSON trees. In addition, Typed JSON allows
 
 In addition to allowing the developer to create object trees and assign properties one-by-one, Typed JSON also supports binding data with the `Data` property. In this way, you can assign database results to populate JSON. The data bindings are two-way bindings: changes to the data object is reflected in the Typed JSON and the changes to the Typed JSON is propagated to the data object.
 
-Server-side view-models leverage the JSON capabilities of Starcounter. You can create a Typed JSON on the server that you send to the client **and** keep on the server. This means that the server can have direct access to the view-model from its controller logic. This is a faster and more secure way to deal with application logic as the client will only deal with presentation logic.
+Server-side view-models leverage the JSON capabilities of Starcounter. You can create a Typed JSON on the server that you send to the client **and** keep on the server. This means that the server can have direct access to the view-model from its controller logic. This is a faster and more secure way to deal with application logic as the client will only deal with view logic.
 
 ## View
 
-Because view-models are shared between the client and server, view rendering is performed on the client. In this way, you get better performance and as the number of users grow, so does the number of CPUs that renders the user interfaces.
+Because view-models are shared between the client and server, it allows for the offset the view rendering to be performed on the client. In this way, you get better performance and as the number of users grow, so does the number of CPUs that renders the user interfaces.
```

Comment:
> it allows for the offset the view rendering to be performed on the client.

This sentence is hard to read. Would something like this work? 

> Because view-models are shared between the client and server, the client can render the view.

I'm not sure if that's what you're trying to say though.

Reply:
There was a grammatical error. I will write: